modules/oadp-pod-volume-backup.adoc

```diff
@@ -8,10 +8,11 @@
 
 Before you start a backup operation by using File System Backup (FSB), you must specify which pods contain a volume that you want to back up. Velero refers to this process as "discovering" the appropriate pod volumes.
 
-Velero supports two approaches for determining pod volumes:
+Velero supports two approaches for determining pod volumes. Use the opt-in or opt-out approach to allow Velero to decide between an FSB, a volume snapshot, or a Data Mover backup. 
 
-* *Opt-in approach*: The opt-in approach requires that you actively indicate that you want to include - _opt-in_ - a volume in a backup. You do this by labelling each pod that contains a volume to be backed up with the name of the volume. When Velero finds a persistent volume (PV), it checks the pod that mounted the volume. If the pod is labelled with the name of the volume, Velero backs up the pod.
-* *Opt-out approach*: With the opt-out approach, you must actively specify that you want to exclude a volume from a backup. You do this by labelling each pod that contains a volume you do not want to back up with the name of the volume. When Velero finds a PV, it checks the pod that mounted the volume. If the pod is labelled with the volume's name, Velero does not back up the pod.
+* *Opt-in approach*: With the opt-in approach, volumes will be backed up using FSB even though the backup spec is using snapshot or DataMover as the default option. Opted-in by using annotations, use Kopia or Restic. 
+
+* *Opt-out approach*: With the opt-out approach, volumes are not backed up using FSB. Volumes, opted out by the addition of annotations, are backed up using a snapshot or Data Mover.
```

Comment:
With the opt-out approach, volumes are backed up using FSB by default. Snapshots or Data Mover will be used on specific volumes opted-out by annotations.